src/SampleWebApp/Controllers/ControllerWithReadWriteActionsController.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SampleWebApp.Controllers
+{
+    public class ControllerWithReadWriteActionsController : Controller
```

Comment:
Huh? This is the same as the empty actions. Normally we'd generate all the CRUD method stubs